lib/pages/caregiver/calendar_page.dart

```diff
@@ -283,7 +284,11 @@ class _CaregiverCalendarPageState extends State<CaregiverCalendarPage> with Tick
 	          onTapped: onChange != null ? () => onChange(item.value, !checked) : null,
 	          isActive: checked
 	        )
-	      )
+	      ),
+				emptyBuilder: (str) => AppHero(
+					icon: Icons.warning,
+					title: AppLocales.of(context).translate('page.caregiverSection.planForm.fields.assignedChildren.emptyList'),
```

Comment:
That's not going to work very well.. "..you can continue creating a plan.."

Reply:
Frick, I forgot I added this second part to this.

Reply:
Will just the first part of the sentence do the trick?
_"You haven't added any child profile to your account yet"_
I'm creating separate text key anyway, Thinking about writing some additional part like "you can't filter right now" or sth.

Reply:
Great - short & generic. You could add the 2nd part above and format similarly:
![image](https://user-images.githubusercontent.com/8643919/104109339-222bc800-52cd-11eb-8365-a9a0762fe45e.png)

Reply:
Currently I'm using a general widget `AppHero` that handles just icon, text and bonus action. Do you have any specific title to add?

Reply:
I was thinking about your "you can't filter right now"

Reply:
![obraz](https://user-images.githubusercontent.com/36575365/104109572-8fd8f380-52cf-11eb-9cae-49e3419b93fd.png)